website/src/components/confetti/index.js

```diff
@@ -0,0 +1,58 @@
+import React, { useRef, useEffect } from 'react';
```

Comment:
this is def no needed but i added a confetti effect as a surprise easter egg at the end of the guide.

Reply:
if there's no objections, i'm happy to request a frontend task and ask the front end team to review this part of the code